src/plugins/projectGalaxy/ProposalSidebar.vue

```diff
@@ -0,0 +1,24 @@
+<script setup>
+import ProjectGalaxyCustomBlock from './components/CustomBlock.vue';
+
+defineProps({
+  space: Object,
+  proposal: Object,
+  results: Object,
+  loadedResults: Boolean,
+  votes: Object,
+  strategies: Object
+});
+</script>
+
+<template>
+  <ProjectGalaxyCustomBlock
+    v-if="space.plugins?.ProjectGalaxy"
```

Comment:
`ProjectGalaxy` must be lowercase `projectGalaxy`. The directory name off the plugin determines the key.